src/java.base/share/classes/java/lang/reflect/AccessFlag.java

```diff
@@ -0,0 +1,274 @@
+/*
+ * Copyright (c) 2021, 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package java.lang.reflect;
+
+import java.lang.annotation.ElementType;
+import java.util.Set;
+import static java.lang.annotation.ElementType.*;
+
+/**
+ * Represents a JVM access flag on a runtime member, such as a
+ * {@linkplain Class class}, {@linkplain Field field}, or {@linkplain
+ * Executable method}.
+ *
+ * <P>JVM access flags are related to, but distinct from Java language
+ * {@linkplain Modifier modifiers}. Some modifiers and access flags
+ * have a one-to-one correspondence, such as {@code public}. In other
+ * cases, some language-level modifiers do <em>not</em> have an access
+ * flag, such as {@code sealed} (JVMS {@jvms 4.7.31}) and some access
+ * flags have no corresponding modifier, such as {@linkplain SYNTHETIC
+ * synthetic}
+ *
+ * <p>The values for the constants representing the access flags are
+ * taken from the tables in sections {@jvms 4.1}, {@jvms 4.4}, {@jvms
+ * 4.5}, and {@jvms 4.7} of <cite>The Java Virtual Machine
+ * Specification</cite>.
+ *
+ * @see java.lang.reflect.Modifier
+ * @see java.compiler/javax.lang.model.element.Modifier
+ * @since 19
+ */
+@SuppressWarnings("doclint:reference") // cross-module link
+public enum AccessFlag {
```

Comment:
I think there is missing SUPER ACC_SUPER  0x0020

Reply:
Note that the presence or absence of `ACC_SUPER` has no effect since **Java 8**, which always treats it as set regardless of the actual contents of the binary class file.

Reply:
For completeness, I think including SUPER is reasonable, even though has been a no-op for some time. (Some time in the future, there could be a class file version aware additions to this enum class.) Well spotted omission.

Reply:
`ACC_SUPER`may be reconed as `ACC_IDENTITY`by Valhalla, so i'm not sure it's a good idea to add ACC_SUPER.

Reply:
As discussed elsewhere in the PR, given the historical existence of ACC_SUPER, I think it is reasonable to leave it all. Also, even if removed later in the release, it would be helpful to validate how overlapping flags can be handled in practice in Valhalla.